src/app/find-help/view-offer-accommodation/view-offer-accommodation.component.ts

```diff
@@ -33,7 +33,7 @@ export class ViewOfferAccommodationComponent implements OnInit {
 
   copyUrl() {
     navigator.clipboard
-      .writeText(this.router.url)
+      .writeText(window.location.origin + this.router.url)
```

Comment:
użyj UrlHelperService, metody basePath(true), ponieważ w tym kodzie nie doda się baseHref -> /ogloszenia/
i przy okazji jest tam błąd:
return absolutePath ? this._origin : this._baseHref;
zamień na:
return absolutePath ? this._absolutePath : this._baseHref;